src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -626,13 +626,15 @@ ad_utility::streams::stream_generator ExportQueryExecutionTrees::
     return binding.dump();
   };
 
+  bool isFirstRow = true;
```

Comment:
This should be the right location. Though I don't know when `getRowIndicies` returns multiple items.